### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by noelfoka**
## Vet clinic database: create animals table

In this branch, I:

- [ ] Create a database named `vet_clinic`

[ ] Create a table named `animals` with the following columns:
    - `id`: integer
    - `name`: string
    - `date_of_birth`: date
    - `escape_attempts`: integer
    - `neutered`: boolean
    - `weight_kg`: decimal
- [ ] Copy the SQL you used in the previous point into the `schema.sql` file in the generated repository.
- [ ] Insert the following data:
    - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
    - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
    - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
    - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
    
Capture

- [ ] Copy the SQL you used in the previous point into the `data.sql` file in the generated repository.
- [ ] Write queries for the following quests:
    - Find all animals whose name ends in "mon".
Capture

    - List the name of all animals born between 2016 and 2019.
Capture

    - List the name of all animals that are neutered and have less than 3 escape attempts.
Capture

    - List the date of birth of all animals named either "Agumon" or "Pikachu".
Capture

    - List name and escape attempts of animals that weigh more than 10.5kg. 
Capture

    - Find all animals that are neutered. 
Capture

    - Find all animals not named Gabumon. 
Capture

    - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg). 
Capture

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 CREATE TABLE animals (
-    name varchar(100)
-);
+	id INTEGER,
+	name varchar,
+	date_of_birth date,
+	escape_attempts INTEGER,
+	neutered boolean,
+	weight_kg decimal
+);
```
Comment (binhussen) [inline schema.sql]: - [Optional] Kindly would you make  `id` column not null, autoincremented, and as primary key to uniquely identify each row in a table 👍

File: README.md [hunk 2]
```diff
@@ -286,6 +185,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (binhussen) [inline README.md:184]: - Kindly would you fix the`mit` broken link for better user accessibility 👍

File: README.md [hunk 3]
```diff
@@ -1,8 +1,6 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+# Vet clinic database
 
+![Alt big picture](/queries.PNG)
```
Comment (binhussen) [inline README.md]: - Kindly fix the broken image or remove it for better user accessibility 👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @noelfoka , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Added animal table ✔️
- Well Structured Query ✔️
- Used Git flow ✔️

## Required Changes ♻️
- Kindly add a screenshot of their query results to the pull request description as mentioned in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md) 👍 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — APPROVED** [review]
# Hi @noelfoka 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Broken links have been removed ✔
- Correct use of Git Flow ✔
- Descriptive PR title with a great screenshot summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**